raspberry_pi_installation.md

```diff
@@ -41,37 +41,29 @@ The RPi installation of Signal K can be broken down in to three distinct steps;
 
 ## Step 1 - Install the Dependencies
 
-Raspbian the LINUX distribution for RPi is based on Debian which has a powerful installation tool called "apt-get", which we will use to install some of the dependencies and tools required. Before you use "apt-get" it is always recommended to update its index of repositories (websites that store LINUX software) and to do that use the following command...
+Raspbian the Linux distribution for RPi is based on Debian which has a powerful installation tool called "apt-get", which we will use to install some of the dependencies and tools required. Before you use "apt-get" it is always recommended to update its index of repositories (websites that store Linux software) and to do that use the following command...
```

Comment:
Everything should refer to apt and not apt-get now?

Reply:
Good point. I guess the cool kids do `apt` instead of `apt-get`. I'll change it everywhere.

Reply:
Updated